noaa/track.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+
+async def sql(es, params):
+    logger = logging.getLogger(__name__)
+    logger.info("Provided params [%s].", params)
+    
+    body = params.get("body")
```

Comment:
The body should be mandatory, thus I propose we retrieve it with `params["body"]` which would fail if the body is not provided.